lib/mongoose_push_web/controllers/healthcheck_controller.ex

```diff
@@ -0,0 +1,119 @@
+defmodule MongoosePushWeb.HealthcheckController do
+  use MongoosePushWeb, :controller
+
+  def send(conn = %Plug.Conn{}, %{}) do
+    stats = :wpool.stats()
+
+    connections =
+      stats
+      |> Enum.map(&extract_connection_info_from_pool/1)
```

Comment:
The coding style remark is that it is considered a bad practise to use pipe only once. Please follow [the style guide](https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines) in future.

Reply:
Good catch, there were more pipes earlier that got removed